src/Symfony/Component/Console/Output/ConsoleOutput.php

```diff
@@ -153,14 +153,18 @@ private function openOutputStream()
             return fopen('php://output', 'w');
         }
 
-        return @fopen('php://stdout', 'w') ?: fopen('php://output', 'w');
+        return \defined('STDOUT') ? \STDOUT : (@fopen('php://stdout', 'w') ?: fopen('php://output', 'w'));
```

Comment:
in absence of test case, a small comment would be nice in case this does fix the issue

Reply:
comment added